index.php

```diff
@@ -0,0 +1,139 @@
+<?php
+// functions
+function previewArray(array $myArray)
+{
+    /* ** Fancy Var dump **
```

Comment:
Doc blocks should be outside the function just before its declared. print_r should not be used, function should return instead
```suggestion
/* ** Fancy Var dump **
* This function receives an array, adds formatting and returns the data in an array   
*
* @Pararm ARRAY - An array is passed in for formatting
*/
function previewArray(array $myArray)
{
    return($myArray);
}
```